insights/parsers/sysconfig.py

```diff
@@ -506,3 +509,39 @@ class VirtWhoSysconfig(SysconfigOptions):
         'A TEST'
     """
     pass
+
+
+@parser(Specs.ifcfg_static_route)
+class IfCFGStaticRoute(SysconfigOptions):
+    """
+    IfCFGStaticRoute is a parser for the static route network interface
+    definition files in ``/etc/sysconfig/network-scripts``.  These are
+    pulled into the network scripts using ``source``, so they are mainly
+    ``bash`` environment declarations of the form **KEY=value**.  These
+    are stored in the ``data`` property as a dictionary.  Quotes surrounding
+    the value
+
+    Because this parser reads multiple files, the interfaces are stored as a
+    list within the parser and need to be iterated through in order to find
+    specific interfaces.
+
+    Sample configuration from a static connection in file ``/etc/sysconfig/network-scripts/rute-test-net``::
+
+        ADDRESS0=10.65.223.0
+        NETMASK0=255.255.254.0
+        GATEWAY0=10.65.223.1
+
+    Examples:
+
+        >>> conn_info.data
+        {'test-net': ['ADDRESS0=10.65.223.0', 'NETMASK0=255.255.254.0', 'GATEWAY0=10.65.223.1']}
+
+    Attributes:
+        data (dict): all the static route cofiguration details
+        static_route_name (str): static route name
+
+    """
+    def parse_content(self, content):
+        self.data = dict()
+        self.static_route_name = self.file_name.split("route-", 1)[1]
+        self.data[self.static_route_name] = content
```

Comment:
I believe the `self.data` should be the parsed content, but not the RAW `content`.

You may want to achieve that by calling the `parser_content` of the superclass as below:

```python
self.static_route_name = self.file_name.split("route-", 1)[1]
super(IfCFGStaticRoute, self).parse_content(content)
```